error-issue/src/issue_builder.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {formatDate} from './utils';
+import {BlameRange, ErrorReport} from './types';
+
+/**
+ * Builds a GitHub issue for a reported error.
+ */
+export class IssueBuilder {
+  private errorId: string;
+  private firstSeen: Date;
+  private dailyOccurrences: number;
+  private message: string;
+  private stack: Array<string>;
+
+  constructor(
+    {errorId, firstSeen, dailyOccurrences, stacktrace}: ErrorReport,
+    private blames: Array<BlameRange>
+  ) {
+    const [message, ...stack] = stacktrace.split('\n');
+    Object.assign(this, {errorId, firstSeen, dailyOccurrences, message, stack});
+  }
+
+  get title(): string {
+    return `🚨 ${this.message}`;
+  }
+
+  get labels(): Array<string> {
+    return ['Type: Error Report'];
+  }
+
+  get bodyDetails(): string {
+    return [
+      'Details',
+      '---',
+      `**Error report:** [link](go/ampe/${this.errorId})`,
```

Comment:
Can you make this a public GCloud link? If somebody doesn't have permission on this they'll get a 401/403, and non-googlers with permissions won't have to jump through hoops to get to the error report :)

Reply:
Punting on this, as I haven't found any way to make accessible a Pantheon project created internally.